### PR TITLE
adding controller logs

### PR DESCRIPTION
**Pull request by anmol372**

### CODE REVIEW

**Review by tanyatukade — APPROVED** [review]
LGTM

File: pkg/controller/acicontainersoperator.go [hunk 1]
```diff
@@ -450,25 +451,28 @@ func (c *Controller) handleConfigMapCreate(newobj interface{}) bool {
 	log.Info("Writing new aci-operator-config configmap")
 	err := c.WriteConfigMap("/tmp/aci-operator.conf", new_config)
```
Comment (anmol372) [inline pkg/controller/acicontainersoperator.go:452]: The Read\WriteConfigMap functions log the error, logged in debug as well.

File: pkg/controller/acicontainersoperator.go [hunk 2]
```diff
@@ -1125,16 +1131,19 @@ func (c *Controller) updatednsOperator() error {
 		scheme := runtime.NewScheme()
 		err := operatorv1.Install(scheme)
 		if err != nil {
+			log.Debugf("Failed to create operator, err: %v", err)
```
Comment (anmol372) [inline pkg/controller/acicontainersoperator.go:1134]: util function logged on return

File: pkg/controller/subnetcache.go [hunk 3]
```diff
@@ -94,6 +95,7 @@ func (cont *AciController) BuildSubnetDnCache(dn string, aciVrfDn string) {
 
 	apicresp, err := cont.apicConn.GetApicResponse(bdUri)
 	if err != nil {
+		cont.log.Debugf("Failed to get APIC response, err: %v", err)
```
Comment (anmol372) [inline pkg/controller/subnetcache.go:98]: GetApicResponse logs the error.

**Review by snaiksat — APPROVED** [review]

**Pull request merged**